src/Compilers/Core/Portable/Symbols/TypeCompareKind.cs

```diff
@@ -10,13 +10,16 @@ namespace Microsoft.CodeAnalysis
     [Flags]
     internal enum TypeCompareKind
     {
-        ConsiderEverything = 0,
+        ConsiderEverything = 0, // PROTOTYPE(NullableReferenceTypes): Does this include comparing nullable modifiers? If not, rename.
```

Comment:
It feels like the comment should include the answer to the question: "Does this include comparing nullable modifiers?" Rather than the question.